docker-compose.integration-postgres.yml

```diff
@@ -13,7 +23,3 @@ services:
       - 6432
     ports:
       - "0.0.0.0:6432:5432"
-    volumes:
-      # NOTE: consider running initialization SQL in Python instead of this
-      # (see "Initialization scripts" section at https://hub.docker.com/_/postgres)
-      - ./data/sql/postgres_example.sql:/docker-entrypoint-initdb.d/postgres_example.sql
```

Comment:
Does this break our postman flow, since the SQL required to create / populate the db is only loaded in with a test fixture? I haven't tested, but when we make integration-env, can we run a successful access request against Postgres?

Reply:
Great spot, this does. I've added this TODO to rectify that before we merge:
![Screenshot 2022-02-18 at 14 30 49](https://user-images.githubusercontent.com/1667340/154749394-14b80ebc-f963-484b-ab3a-0bfcba76e444.png)